app/assets/js/app/modules/totalliser.js

```diff
@@ -0,0 +1,73 @@
+import domready from './domready'
+import getTransitionEndEvent from '../helpers/transitionend'
+
+const transitionendEvent = getTransitionEndEvent()
+
+domready(() => {
+  const breakdownFields = document.querySelectorAll('input:not(.js-input-calculated)')
+  const totalField = document.querySelector('.js-input-calculated')
+  const fieldSet = document.querySelector('.question__fieldset')
```

Comment:
Please only use specific 'js-' classes as selectors in js, never classes used for styling.